jobs/utils/snowflake.py

```diff
@@ -197,20 +207,34 @@ def gdf_to_snowflake(
         for c, dtype in gdf.dtypes.to_dict().items():
             if type(dtype) == geopandas.array.GeometryDtype:
                 if epsg == WGS84:
-                    cols.append(f'TO_GEOGRAPHY("{c}") AS "{c}"')
+                    cols.append(
+                        f'{"" if strict_geometries else "TRY_"}TO_GEOGRAPHY("{c}") AS "{c}"'
+                    )
                 else:
-                    cols.append(f'TO_GEOMETRY("{c}", {epsg}) AS "{c}"')
+                    cols.append(
+                        f'{"" if strict_geometries else "TRY_"}TO_GEOMETRY("{c}", {epsg}) AS "{c}"'
+                    )
             else:
                 cols.append(f'"{c}"')
 
-        # Create the final table, selecting from the temp table.
-        sql = f'''CREATE OR REPLACE TABLE "{database}"."{schema}"."{table_name}"'''
-        if cluster:
-            sql = sql + f"\nCLUSTER BY ({','.join(cluster_names)})"
-        sql = (
-            sql
-            + f'''\nAS SELECT \n{",".join(cols)} \nFROM "{database}"."{schema}"."{tmp_table}"'''
-        )
+        if overwrite:
+            # Create the final table, selecting from the temp table.
+            sql = f'''CREATE OR REPLACE TABLE "{database}"."{schema}"."{table_name}"'''
+            if cluster:
+                sql = sql + f"\nCLUSTER BY ({','.join(cluster_names)})"
+            sql = (
+                sql
+                + f'''\nAS SELECT \n{",".join(cols)} \nFROM "{database}"."{schema}"."{tmp_table}"'''
+            )
+        else:
+            if cluster:
+                raise ValueError(
+                    "Clustering on existing table (overwrite=False) not supported"
+                )
+            sql = (
+                f'''INSERT INTO "{database}"."{schema}"."{table_name}"'''
+                f'''\nSELECT \n{",".join(cols)} \nFROM "{database}"."{schema}"."{tmp_table}"'''
+            )
```

Comment:
In the past we were overwriting every time. But with this dataset being split into different quadkeys, we have to have logic for appending to an existing table.

Reply:
By "in the past" are you reference to the US Building Footprints data?

Reply:
Yes, that and other single-file geospatial datasets we've loaded (e.g., all the tiger ones)

Reply:
Got it and we'll still overwrite for those, but append for the Global dataset?

Reply:
Correct, it's controlled using the `overwrite` parameter.